app/settings.py

```diff
@@ -17,22 +17,40 @@
 BASE_DIR = Path(__file__).resolve().parent.parent
 
 
+PULP_DEPLOYMENT = os.environ.get("PULP_DEPLOYMENT")
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get("APP_KEY")
+SECRET_KEY = os.environ["APP_KEY"]
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = PULP_DEPLOYMENT is None
```

Comment:
Can we move this down into the `if/elif/else` section too please? I think it would be really nice to super clearly see what it's value is in each case.